include/boost/http_proto/serializer.hpp

```diff
@@ -147,6 +145,36 @@ class BOOST_SYMBOL_VISIBLE
 
     //--------------------------------------------
 
+    /** Return a new stream for this serializer.
+
+        After the serializer is destroyed, `reset` is called,
+        or `is_done` returns true, the only valid operation
+        on the stream is destruction.
+
+        A stream may be used to invert the flow of control
+        when the caller is supplying body data as a series
+        of buffers.
+
+        @code{.cpp}
+        std::string msg = "Hello, world!";
+        std::size_t buf_size = 16 * 1024;
+        http_proto::serializer sr(buf_size);
```

Comment:
these code snippets should be as short as possible you don't need `buf_size` as a variable just put the number in the function call argument list.

Reply:
Good idea.
Most importantly, this didn't even compile.